app/src/views/features/rules/RuleEditor/components/Header/index.js

```diff
@@ -59,6 +76,21 @@ const Header = ({ mode, location, currentlySelectedRuleData, currentlySelectedRu
             <Col>
               <Status location={location} />
             </Col>
+            {isRuleGroupDisabled && (
+              <Tooltip title="This rule won't execute because its parent group is disabled. Enable the group to run this rule.">
+                <Col className="rule-editor-header-disabled-group-warning">
+                  <WarningOutlined className="icon__wrapper" />
+                  Group is disabled.{" "}
+                  <Link
+                    onClick={() =>
+                      recordStatusToggleAction(normalizeRecord(allRecordsMap[currentlySelectedRuleData.groupId]))
+                    }
+                  >
+                    Enable now
+                  </Link>
```

Comment:
This shouldn't be a `Link` as this does not navigate to other route, but performs some action. We can use the antd `Button` with `link` type here.